libraries/chain/controller.cpp

```diff
@@ -4367,8 +4366,10 @@ struct controller_impl {
             break;
          }
       }
-      dlog("removed ${n} expired transactions of the ${t} input dedup list, pending block time ${pt}",
-           ("n", num_removed)("t", total)("pt", now));
+      if (!replaying && total > 0) {
```

Comment:
I like this removal. Was thinking about removing this spamming `removed 0 expired transactions...` too.